docs/configuration.rst

```diff
@@ -268,6 +268,22 @@ Expected version is **3.2** or higher.
 .. _`ElastiCache Redis Parameter Group`: http://docs.aws.amazon.com/AmazonElastiCache/latest/UserGuide/ParameterGroups.Redis.html#ParameterGroups.Redis.3-2-4
 
 
+Redis Socket Timeouts
+=====================
+
+There are two Redis connections. The "Redis Cache" and the "Redis Store".
+These have both have the same defaults for
+``SOCKET_CONNECT_TIMEOUT`` (1 second) and ``SOCKET_TIMEOUT`` (2 seconds).
+
+The environment variables and their defaults are listed below:
+
+.. code-block:: shell
+
+    DJANGO_REDIS_SOCKET_CONNECT_TIMEOUT=1
+    DJANGO_REDIS_SOCKET_TIMEOUT=2
```

Comment:
For consistency, maybe these should be prefixed with DJANGO_REDIS_CACHE

Reply:
Maybe but I use a similar pattern here: https://github.com/mozilla-services/tecken/blob/35739beecdb84aeaa92832cd645f73b692bef10c/tecken/settings.py#L272-L273

The redis for caching is `REDIS_URL` whereas the Redis for the LRU Store is `REDIS_STORE_URL`.